bool erase
bool current_only
---
Session[] sessions
